lua_modules/cohelper/cohelper.lua

```diff
@@ -0,0 +1,27 @@
+--[[  A coroutine Helper   T. Ellison,  June 2019
+
+This version of couroutine helper demonstrates the use of corouting within
+NodeMCU execution to split structured Lua code into smaller tasks
+
+]]
+--luacheck: read globals node
```

Comment:
This won't be needed - I forgot about the `task` functions when creating luacheck config and I fixed it in  https://github.com/nodemcu/nodemcu-firmware/pull/2846/commits/990515a2f29afb50fd038f7e49304b7b3ef569c6 in #2846

Reply:
Done